### DISCUSSION TITLE
Issue with steaming video on Enterprise Multiserver setup

### DISCUSSION THREAD

**Discussion by yonescat** [mozilla/hubs#3896] [Hubs Cloud Questions] [answered]
Hi Hubs,

Please help, we have a single server and enterprise multi-server setup. We have an event on Saturday, less and 48hrs.

we're using a Twitch feed but we've noticed on the enterprise version (2 server deployment), the steam freezes every 3-5 sec then goes again! the same stream is running ok on our single server setup!

Our two deployments are here, https://my.yohubs.app/ (personal single server), https://live.yohubs.app/ (enterprise multiserver),

I think video buffer jumps between the two servers.

Any assistance would be appreciated.

Yonescat Team
AKA Heman

**Comment by robinkwilson** [comment, marked as answer]
Hello! Can you give me more information around this:

Any errors in the developer tools console? 
Any screenshots? 
How large is this video? 
Does it happen with a smaller video?
Are you using a service for video streaming?
How many people are connected to the instance at one time? 

Any other information would help! Thanks!

**Reply by robinkwilson** [reply]
Marking as answered for now, please unmark as answered when you include more information!
